source/help/messaging/organizing-conversations.rst

```diff
@@ -0,0 +1,72 @@
+Organizing Conversations using Collapsed Reply Threads (Beta)
+=============================================================
+
+Threads are a key part of the messaging experience in Mattermost. Threads organize conversations within channels, and threads enable users to discuss topics without adding noise to channels or Direct Messages. 
+
+Collapsed Reply Threads offers an enhanced experience for users participating in conversations by:
+
+- Improving channel details to find, follow, and resume conversations more easily.
+- Prioritizing how conversation threads are displayed in Mattermost.
+
+.. image:: ../../images/collapsed-reply-threads.gif
+  :alt: Organize conversations using Collapsed Reply Threads.
+
+Your System Admin must enable the Collapsed Reply Threads feature in the System Console. See our `Configuration Settings <https://docs.mattermost.com/administration/config-settings.html#collapsed-reply-threads-beta>`__ documentation for details. You may additionally need to enable this feature for your Mattermost account by going to **Account Settings > Display > Collapsed Reply Threads (Beta)**.
+
+Known Issues
+------------
+
+.. important::
+
+    Collapsed Reply Threads is available as a beta release from Mattermost v5.37. You may experience minor issues with this beta functionality. 
```

Comment:
Collapsed Reply Threads are available as an early beta in Mattermost Server v5.37 and later. It's expected that you may experience bugs as we stabilize the feature. In particular, please be aware of these known issues: